sensor/common/compliance/service.go

```diff
@@ -21,13 +22,15 @@ type Service interface {
 
 	Output() chan *compliance.ComplianceReturn
 	AuditEvents() chan *sensor.AuditEvents
+	NodeScansV2() <-chan *storage.NodeScanV2
```

Comment:
Maybe just `NodeScans`?
```suggestion
	NodeScans() <-chan *storage.NodeScanV2
```

Reply:
In regards to V2: You're absolutely right about V1 not being a real NodeScan ([src](https://github.com/stackrox/stackrox/pull/3533#discussion_r1010714715)), but as the existing feature is called NodeScan in every piece of currently used code, it might be very confusing to introduce a completely different thing under the same name, right?

Reply:
I will make a proposal to clear up the naming and their meaning, as I see that it makes even more problems when navigating though the code of Central and Scanner.

Reply:
I don't mind keeping V2. In this case I'd vote to have V2 more consistently. There are some spots in this PR where it could be (re-)added.

Reply:
Follow-up to the naming discussion: https://github.com/stackrox/stackrox/pull/3755